opentimelineio/adapters/cmx_3600.py

```diff
@@ -71,6 +72,15 @@ class EDLParseError(otio.exceptions.OTIOError):
 }
 
 
+# Currently, the 'style' argument determines
+# the comment string for the media reference.
+#   'avid': '* FROM CLIP:' (default)
+#   'nucoda': '* FROM FILE:'
```

Comment:
Can you add some advice to future developers about adding new styles? Something along the lines of "When adding a new style, please make sure to add a test to verify that you are getting the results you want without disrupting the existing styles."

Reply:
Done!